docs/site/src/components/PlatformTabs.astro

```diff
@@ -44,11 +44,53 @@ const populatedSlots = Object.fromEntries(
   }
   {
     populatedSlots["ios"] && (
-      <TabItem label="iOS" icon="apple">
+      <TabItem label="iOS" icon="apple" data-lang="ios">
         <div class="card">
           <slot name="ios" />
         </div>
       </TabItem>
     )
   }
 </Tabs>
+
+<script>
```

Comment:
Is there a way we can write this as standalone JS and inject it? Just worried about the maintenance around this

Reply:
Gonna leave it in since the logic is coupled with the component itself